_data/Aquahamzah/Aquahamzah.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Aquahamzah
+institution: City, University of London
+profile_pic: aquahamzah.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `aquahamzah.jpg` to `aquahamzah.png` in the markdown file.
2. Rename `aquahamzah.png` to `aquahamzah.jpg`.
I'd suggest you make the change in the markdown file!

Reply:
Done this! My apologies again, forgot to change the .md file to match the new file upload for my profile picture. :)